app/src/main/java/com/github/pockethub/ui/WebView.java

```diff
@@ -17,12 +17,15 @@
 
 import android.content.Context;
 import android.util.AttributeSet;
+import android.view.MotionEvent;
 
 /**
  * Web view extension with scrolling fixes
  */
 public class WebView extends android.webkit.WebView {
```

Comment:
We use it at two places and both of those could be changed to used the standard WebView since they don't require any additional functions